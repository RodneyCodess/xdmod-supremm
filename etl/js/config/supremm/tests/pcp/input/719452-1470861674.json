{
   "_id":"719452-1470861674",
   "summarization":{
      "srcdir":"/dev/shm/supremm_test/jellystone/719452",
      "version":"1.0.5",
      "complete":false,
      "mergetime":0.040428876876831055,
      "elapsed":0.01883697509765625
   },
   "procDump":{
      "unconstrained":[

      ],
      "constrained":[

      ],
      "cpusallowed":{
         "node1":{
            "node":"node1223",
            "error":1
         },
         "node0":{
            "node":"node1255",
            "error":1
         },
         "node2":{
            "node":"node5489",
            "error":1
         }
      }
   },
   "network":{
      "total":{
         "in-bytes":{
            "std":3353346.705018187,
            "med":6676762,
            "skw":0.706618844462775,
            "min":6593709,
            "max":12442957,
            "cov":0.3912368321740128,
            "cnt":3,
            "krt":-1.4999999999999991,
            "avg":8571142.666666666
         },
         "out-packets":{
            "std":5912.08307902835,
            "med":47806,
            "skw":-0.18532627670032203,
            "min":41013,
            "max":52791,
            "cov":0.12524715229916708,
            "cnt":3,
            "krt":-1.4999999999999991,
            "avg":47203.333333333336
         },
         "out-bytes":{
            "std":9125494.75780119,
            "med":3836251,
            "skw":0.7030482380437021,
            "min":3184265,
            "max":19305990,
            "cov":1.0398829329423194,
            "cnt":3,
            "krt":-1.4999999999999996,
            "avg":8775502
         },
         "in-packets":{
            "std":3340.717038800702,
            "med":79962,
            "skw":-0.6438538977767619,
            "min":74707,
            "max":80908,
            "cov":0.04254299492905549,
            "cnt":3,
            "krt":-1.499999999999995,
            "avg":78525.66666666667
         }
      }
   },
   "acct":{
      "uid":null,
      "exit_status":null,
      "resource_id": 1,
      "ncpus": 90,
      "host_list":[
         "node1223",
         "node1255",
         "node5489"
      ],
      "id":"719452",
      "local_job_id": 719452,
      "group":"Unknown",
      "reqmem":null,
      "eligible":null,
      "exit_code":null,
      "submit": 1470858653,
      "gid":null,
      "reqcpus":null,
      "nodes": 6,
      "start_time": 1470858850,
      "jobname":"namd2-gpu",
      "user":"rbunting",
      "account":"",
      "partition":"regular",
      "timelimit":null,
      "end_time": 1470861674
   },
   "cpu":{
      "jobcpus":{
         "error":5
      },
      "nodecpus":{
         "all":{
            "cnt":3
         },
         "iowait":{
            "std":0.020066387964796352,
            "med":0.15249463603731356,
            "skw":0.5868279654153268,
            "min":0.144628134370615,
            "max":0.18264317493846977,
            "cov":0.12547610866987308,
            "cnt":3,
            "krt":-1.4999999999999987,
            "avg":0.15992198178213277
         },
         "system":{
            "cnt":3,
            "avg":0
         },
         "idle":{
            "std":0.006268035418458056,
            "med":0.004702145029703117,
            "skw":0.7032876723480702,
            "min":0.004267732439395718,
            "max":0.015334974139891227,
            "cov":0.7736770648876854,
            "cnt":3,
            "krt":-1.4999999999999996,
            "avg":0.008101617202996687
         },
         "user":{
            "std":0.02627132653372262,
            "med":0.8428032189329834,
            "skw":-0.6285184018378394,
            "min":0.802021850921639,
            "max":0.8511041331899892,
            "cov":0.03157700927775841,
            "cnt":3,
            "krt":-1.4999999999999958,
            "avg":0.8319764010148706
         },
         "irq":{
            "cnt":3,
            "avg":0
         }
      }
   },
   "nodememory":{
      "maxused":{
         "std":3109.342749413987,
         "med":33529241,
         "skw":0.5912403688162158,
         "min":33528045,
         "max":33533928,
         "cov":0.00009273203769309277,
         "cnt":3,
         "krt":-1.5000000000011566,
         "avg":33530404.666666668
      },
      "used":{
         "std":847.3723939085656,
         "med":33528835.875,
         "skw":-0.2672145733025726,
         "min":33527806.5,
         "max":33529487.125,
         "cov":0.000025273039079664526,
         "cnt":3,
         "krt":-1.5000000000019185,
         "avg":33528709.833333332
      },
      "free":{
         "std":847.3723939085656,
         "med":25596.125,
         "skw":0.2672145733079516,
         "min":24944.875,
         "max":26625.5,
         "cov":0.03294327437068802,
         "cnt":3,
         "krt":-1.5000000000000022,
         "avg":25722.166666666668
      },
      "maxfree":{
         "std":1477.6868184203759,
         "med":27998,
         "skw":-0.6776305011575596,
         "min":25593,
         "max":28283,
         "cov":0.05414491114714229,
         "cnt":3,
         "krt":-1.5000000000000022,
         "avg":27291.333333333332
      }
   }
}
