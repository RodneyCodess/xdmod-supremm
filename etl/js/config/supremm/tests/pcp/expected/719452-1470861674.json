{
    "local_job_id": {
        "value": "719452",
        "error": 0
    },
    "name": {
        "value": "namd2-gpu",
        "error": 0
    },
    "resource_name": {
        "value": "pcp",
        "error": 0
    },
    "resource_id": {
        "value": 1,
        "error": 0
    },
    "organization_id": {
        "value": 1,
        "error": 0
    },
    "account": {
        "value": "",
        "error": 0
    },
    "username": {
        "value": "rbunting",
        "error": 0
    },
    "cwd": {
        "value": "NA",
        "error": 2
    },
    "executable": {
        "value": "NA",
        "error": 8
    },
    "application": {
        "value": "NA",
        "error": 8
    },
    "exit_status": {
        "value": "NA",
        "error": 0
    },
    "datasource": {
        "value": "pcp",
        "error": 0
    },
    "granted_pe": {
        "value": 90,
        "error": 0
    },
    "queue": {
        "value": "regular",
        "error": 0
    },
    "requested_nodes": {
        "value": 6,
        "error": 0
    },
    "hosts": {
        "value": [
            "node1223",
            "node1255",
            "node5489"
        ],
        "error": 0
    },
    "nodes": {
        "value": 6,
        "error": 0
    },
    "shared": {
        "value": 0,
        "error": 0
    },
    "cores": {
        "value": 90,
        "error": 0
    },
    "cores_avail": {
        "value": 0,
        "error": 1
    },
    "submit_time_ts": {
        "value": 1470858653,
        "error": 0
    },
    "eligible_time_ts": {
        "value": null,
        "error": 0
    },
    "start_time_ts": {
        "value": 1470858850,
        "error": 0
    },
    "end_time_ts": {
        "value": 1470861674,
        "error": 0
    },
    "wall_time": {
        "value": 2824,
        "error": 0
    },
    "requested_wall_time": {
        "value": null,
        "error": 2
    },
    "wait_time": {
        "value": 197,
        "error": 0
    },
    "cpu_time": {
        "value": 254160,
        "error": 0
    },
    "node_time": {
        "value": 16944,
        "error": 0
    },
    "cpu_idle": {
        "value": 0.008101617202996687,
        "error": 0
    },
    "cpu_system": {
        "value": 0,
        "error": 0
    },
    "cpu_user": {
        "value": 0.8319764010148706,
        "error": 0
    },
    "error": {
        "value": null,
        "error": 2
    },
    "flops": {
        "value": null,
        "error": 8
    },
    "flops_cov": {
        "value": null,
        "error": 8
    },
    "cpiref": {
        "value": null,
        "error": 8
    },
    "cpiref_cov": {
        "value": null,
        "error": 8
    },
    "catastrophe": {
        "value": null,
        "error": 2
    },
    "cpldref": {
        "value": null,
        "error": 8
    },
    "cpldref_cov": {
        "value": null,
        "error": 8
    },
    "mem_transferred": {
        "value": null,
        "error": 8
    },
    "mem_transferred_cov": {
        "value": null,
        "error": 8
    },
    "cpu_user_imbalance": {
        "value": 5.766895066575095,
        "error": 0
    },
    "cpu_user_cv": {
        "value": 0.03157700927775841,
        "error": 0
    },
    "node_cpu_idle": {
        "value": 0.008101617202996687,
        "error": 0
    },
    "energy": {
        "value": null,
        "error": 8
    },
    "max_power": {
        "value": null,
        "error": 8
    },
    "memory_used": {
        "value": null,
        "error": 8
    },
    "memory_used_cov": {
        "value": null,
        "error": 8
    },
    "max_memory": {
        "value": null,
        "error": 8
    },
    "mem_used_including_os_caches": {
        "value": null,
        "error": 8
    },
    "mem_used_including_os_caches_cov": {
        "value": null,
        "error": 8
    },
    "ib_rx_bytes": {
        "value": null,
        "error": 8
    },
    "block_sda_wr_ios": {
        "value": null,
        "error": 2
    },
    "block_sda_wr_bytes": {
        "value": null,
        "error": 2
    },
    "block_sda_wr_bytes_cov": {
        "value": null,
        "error": 8
    },
    "block_sda_rd_ios": {
        "value": null,
        "error": 2
    },
    "block_sda_rd_bytes": {
        "value": null,
        "error": 2
    },
    "block_sda_rd_bytes_cov": {
        "value": null,
        "error": 8
    },
    "netdrv_gpfs_rx": {
        "value": null,
        "error": 2
    },
    "netdrv_gpfs_rx_cov": {
        "value": null,
        "error": 8
    },
    "netdrv_gpfs_rx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_gpfs_tx": {
        "value": null,
        "error": 2
    },
    "netdrv_gpfs_tx_cov": {
        "value": null,
        "error": 8
    },
    "netdrv_gpfs_tx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_rx": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_rx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_rx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_tx": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_tx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_isilon_tx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_rx": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_rx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_rx_msgs": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_tx": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_tx_cov": {
        "value": null,
        "error": 2
    },
    "netdrv_panasas_tx_msgs": {
        "value": null,
        "error": 2
    },
    "netdir_home_read": {
        "value": null,
        "error": 2
    },
    "netdir_home_write": {
        "value": null,
        "error": 2
    },
    "netdir_projects_read": {
        "value": null,
        "error": 2
    },
    "netdir_projects_write": {
        "value": null,
        "error": 2
    },
    "netdir_util_read": {
        "value": null,
        "error": 2
    },
    "netdir_util_write": {
        "value": null,
        "error": 2
    },
    "net_eth0_rx": {
        "value": null,
        "error": 8
    },
    "net_eth0_rx_cov": {
        "value": null,
        "error": 8
    },
    "net_eth0_rx_packets": {
        "value": null,
        "error": 8
    },
    "net_eth0_tx": {
        "value": null,
        "error": 8
    },
    "net_eth0_tx_cov": {
        "value": null,
        "error": 8
    },
    "net_eth0_tx_packets": {
        "value": null,
        "error": 8
    },
    "net_ib0_rx": {
        "value": null,
        "error": 8
    },
    "net_ib0_rx_packets": {
        "value": null,
        "error": 8
    },
    "net_ib0_tx": {
        "value": null,
        "error": 8
    },
    "net_ib0_tx_packets": {
        "value": null,
        "error": 8
    },
    "gpus": {
        "value": 0,
        "error": 8
    },
    "gpu_time": {
        "value": null,
        "error": 8
    },
    "gpu_usage": {
        "value": null,
        "error": 8
    },
    "gpu_energy": {
        "value": null,
        "error": 2
    },
    "gpu_max_power": {
        "value": null,
        "error": 2
    },
    "gpu0_nv_mem_used": {
        "value": null,
        "error": 8
    },
    "gpu0_nv_utilization": {
        "value": null,
        "error": 8
    }
}
